### DISCUSSION TITLE
How do I setup on bungee?

### DISCUSSION THREAD

**Discussion by HarryPlays2021** [nickuc/OpeNLogin#49] [Q&A] [answered]
How do i setup this plugin on my bungeecord server?

**Comment by nickuc** [comment, marked as answer]
Hi, @HarryPlays2021!

Unfortunately, **OpeNLogin** doesn't support BungeeCord yet.

For now, I recommend that you use **nLogin** and configure BungeeGuard: https://www.nickuc.com/docs/bungeeguard
